### DISCUSSION TITLE
Show my (own) log option/icon in logs header

### DISCUSSION THREAD

**Discussion by LettaLet** [2Abendsegler/GClh#2090] [Questions]
It would be nice if there was an icon in the logs header that shows me my (own) log.

**Comment by 2Abendsegler** [comment]
Hi,

I think a cacher colleague already suggested this in his [post](https://github.com/2Abendsegler/GClh/discussions/2086#discussion-3934539) in the second section of text. 
Is it that what you want?

**Reply by LettaLet** [reply]
I actually mean a shortcut (icon) to my log.

![header](https://user-images.githubusercontent.com/101531249/158084646-79416061-a38f-475c-a145-9130bb66b5ee.JPG)

**Comment by 2Abendsegler** [comment]
I must have gotten totally wrong on this topic. 🤭

The function already exists elsewhere. Check out the VIP list. You should see icons for your logs there. Click on an icon.

**Reply by LettaLet** [reply]
I'm sorry but I can't find that function in my VIP list. By the way i'm not the cache owner! Just a person who ever logged it.

**Reply by 2Abendsegler** [reply]
You must mark yourself as a VIP. For me, for example, it looks like this. If I click the log icon behind my name, then I get to my log.

![grafik](https://user-images.githubusercontent.com/22332216/158130249-c5577a53-0ba2-421b-8ffc-90699df3dad2.png)
